azkaban-exec-server/src/main/java/azkaban/execapp/FlowRunnerManager.java

```diff
@@ -204,6 +205,23 @@ public FlowRunnerManager(final Props props,
             getClass().getClassLoader());
   }
 
+  /**
+   * Setting the gid bit on the execution directory forces all files/directories created within
+   * the directory to be a part of the azkaban group. Then, when users create their own files,
```

Comment:
The service account name may not be azkaban. Consider using a more generic term.